pkgs/top-level/python-packages.nix

```diff
@@ -437,6 +439,8 @@ in {
 
   pymatgen = callPackage ../development/python-modules/pymatgen { };
 
+  PyMsgBox = callPackage ../development/python-modules/PyMsgBox { };
```

Comment:
I'd prefer using `pymsgbox`, also as dir name.

Reply:
Yes, I can change the dir name. How about adding below that:
```
pymsgbox = PyMsgBox;
```
I've heard the package name should be equal to the name in PyPI, so that's why I thought it should be available at least as `PyMsgBox`.

Reply:
I would also prefer if it matches. We should have a policy on that, so we do this consistently for python packages.

Reply:
How about a policy to have both the PyPI name and a lowercase alias?

Reply:
I've written this before several times. We prefer to use [normalized names](https://www.python.org/dev/peps/pep-0503/#normalized-names) throughout, because that matches well with the naming we use throughout Nixpkgs. The `pname` to `fetchPypi` however has to match to the filename used because PyPI does not translate that part of the url. If for `fetchPypi` the name cannot be normalized, then it may be more convenient to use the same one for `buildPython*`.

Reply:
Upstream issue: https://github.com/pypa/warehouse/issues/2537

Reply:
Ok. lets write this paragraph down: https://github.com/NixOS/nixpkgs/pull/45822#issue-212224967

Reply:
So I should normalize `eth-hash` to `eth_hash` and similarly for those other `eth-*` packages?

Reply:
So I should rename `backports_shutil_which` to `backports-shutil-which`, right?

Reply:
Yes